nixos/modules/system/boot/kernel.nix

```diff
@@ -21,6 +23,11 @@ in
 
     boot.kernelPackages = mkOption {
       default = pkgs.linuxPackages;
+      apply = kernelPackages: kernelPackages.override (self: super: {
```

Comment:
Oh, yikes. Yes -- thanks for catching that. Didn't catch that after the renaming `override` to `extend`.